src/commands/publish.ts

```diff
@@ -399,9 +375,15 @@ async function handleReleaseBranch(
 export async function publishMain(argv: PublishOptions): Promise<any> {
   logger.debug('Argv:', JSON.stringify(argv));
   checkMinimalConfigVersion();
-  checkPrerequisites();
+  // TODO (kmclb): pull the names of the necessary env vars out of config once
+  // there are more options than just Zeus
+  checkEnvForPrerequisite({ name: 'ZEUS_API_TOKEN', legacyName: 'ZEUS_TOKEN' });
+  // We currently need ZEUS_TOKEN set for zeus-sdk to work properly
+  if (!process.env.ZEUS_TOKEN) {
+    process.env.ZEUS_TOKEN = process.env.ZEUS_API_TOKEN;
+  }
```

Comment:
We can probably move this block to the Zeus status/artifact provider constructors already.

Reply:
Agreed, but will do that in an artifact-provider-themed PR in the near future.